assets/js/Ioda/pages/dashboard/DashboardTab.js

```diff
@@ -16,62 +16,63 @@ import { asn } from "./DashboardConstants";
 import { AreaChartOutlined, GlobalOutlined } from "@ant-design/icons";
 import { getSecondsAsErrorDurationString } from "../../utils/timeUtils";
 
-class DashboardTab extends Component {
-  constructor(props) {
-    super(props);
-    this.config = React.createRef();
+const DashboardTab = (props) => {
+  console.log(props);
 
-    this.state = {
-      mounted: false,
-    };
-  }
+  const {
+    eventDataProcessed,
+    topoData,
+    topoScores,
+    handleEntityShapeClick,
+    activeTabType,
+    until,
+    from,
+    totalOutages,
+    type,
+    tabCurrentView,
+    handleTabChangeViewButton,
+    summaryDataRaw,
+    genSummaryTableDataProcessed,
+    summaryDataProcessed
+  } = props;
 
-  componentDidMount() {
-    this.setState({ mounted: true });
-  }
 
-  componentWillUnmount() {
-    this.setState({ mounted: false });
-  }
+    const config = React.createRef();
+    const [mounted, setMounted] = useState(false);
 
-  componentDidUpdate(prevProps) {
-    if (!this.state.mounted) {
-      return;
-    }
+  useEffect(() => {
+    setMounted(true);
+  }, []);
```

Comment:
We shouldn't need this, since `useEffect` with no dependencies will only run on component mount. Also, from what I can tell, this `mounted` state isn't being used anywhere to begin with.

Reply:
Yes, I will remove this state.